tests/rustworkx_tests/digraph/test_bipartite.py

```diff
@@ -0,0 +1,59 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import rustworkx
+
+
+class TestBipartite(unittest.TestCase):
```

Comment:
Can you add some test cases using generators:
* `rustworkx.generators.directed_grid_graph` for any `(n, m)` for bipartite case
* `rustworkx.generators.directed_cycle_graph` with odd number of vertices for non bipartite case

Reply:
Done in: https://github.com/Qiskit/rustworkx/pull/1002/commits/0b94c19b68cde43a509394b955046950bd98d9ea